<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QMainWindow" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>941</width>
    <height>578</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: 0 0 0 0 stretch stretch;
background-color: rgb(206, 232, 248)</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="2">
     <widget class="QScrollBar" name="scrlwrmprteam">
      <property name="minimumSize">
       <size>
        <width>201</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(101, 148, 196);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>5</number>
      </property>
      <property name="value">
       <number>3</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="11" column="4">
     <widget class="QCheckBox" name="infinitegrenade">
      <property name="text">
       <string>oo</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="13" column="7">
     <widget class="QScrollBar" name="scrljet">
      <property name="minimumSize">
       <size>
        <width>201</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(101, 148, 196);
color: rgb(0, 0, 0);
</string>
      </property>
      <property name="value">
       <number>2</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="11" column="3">
     <widget class="QLCDNumber" name="lcdNumber_5">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(72, 129, 189)</string>
      </property>
     </widget>
    </item>
    <item row="15" column="3">
     <widget class="QLCDNumber" name="lcdNumber_9">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(72, 129, 189)</string>
      </property>
      <property name="value" stdset="0">
       <double>5.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="11" column="7">
     <widget class="QScrollBar" name="scrlholy">
      <property name="minimumSize">
       <size>
        <width>201</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(101, 148, 196);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="value">
       <number>2</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="11" column="6">
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>HOLY GRENADE</string>
      </property>
     </widget>
    </item>
    <item row="13" column="3">
     <widget class="QLCDNumber" name="lcdNumber_7">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(72, 129, 189)</string>
      </property>
      <property name="value" stdset="0">
       <double>3.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="15" column="6">
     <widget class="QLabel" name="label_23">
      <property name="text">
       <string>MONSTER BOMB</string>
      </property>
     </widget>
    </item>
    <item row="12" column="3">
     <widget class="QLCDNumber" name="lcdNumber_6">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(72, 129, 189)</string>
      </property>
      <property name="value" stdset="0">
       <double>10.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="14" column="3">
     <widget class="QLCDNumber" name="lcdNumber_8">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(72, 129, 189)</string>
      </property>
      <property name="value" stdset="0">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QLCDNumber" name="lcdNumber">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(72, 129, 189)</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>7</number>
      </property>
      <property name="midLineWidth">
       <number>1</number>
      </property>
      <property name="value" stdset="0">
       <double>3.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="10" column="8">
     <widget class="QLCDNumber" name="lcdNumber_12">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(72, 129, 189)</string>
      </property>
      <property name="value" stdset="0">
       <double>10.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="7" column="1" colspan="8">
     <widget class="Line" name="line_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0" rowspan="2">
     <widget class="Line" name="line_9">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="10" column="3">
     <widget class="QLCDNumber" name="lcdNumber_4">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(72, 129, 189)</string>
      </property>
     </widget>
    </item>
    <item row="10" column="4">
     <widget class="QCheckBox" name="infinitebazooka">
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>oo</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="15" column="8">
     <widget class="QLCDNumber" name="lcdNumber_10">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(72, 129, 189)</string>
      </property>
      <property name="value" stdset="0">
       <double>2.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="14" column="6">
     <widget class="QLabel" name="label_22">
      <property name="text">
       <string>ARMAGADEON</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QLCDNumber" name="lcdNumber_2">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(72, 129, 189)</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="value" stdset="0">
       <double>25.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="11" column="8">
     <widget class="QLCDNumber" name="lcdNumber_14">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(72, 129, 189)</string>
      </property>
      <property name="value" stdset="0">
       <double>2.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="4" column="8">
     <widget class="QLCDNumber" name="lcdNumber_3">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(72, 129, 189)</string>
      </property>
      <property name="value" stdset="0">
       <double>30.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="12" column="7">
     <widget class="QScrollBar" name="scrldynamite">
      <property name="minimumSize">
       <size>
        <width>201</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(101, 148, 196);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="value">
       <number>10</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="10" column="5" rowspan="7">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="23" column="1" colspan="8">
     <widget class="QPushButton" name="mainmenu">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>MAIN MENU</string>
      </property>
     </widget>
    </item>
    <item row="13" column="8">
     <widget class="QLCDNumber" name="lcdNumber_16">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(72, 129, 189)</string>
      </property>
      <property name="value" stdset="0">
       <double>2.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="14" column="7">
     <widget class="QScrollBar" name="scrlarma">
      <property name="minimumSize">
       <size>
        <width>201</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(101, 148, 196);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="value">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="12" column="8">
     <widget class="QLCDNumber" name="lcdNumber_15">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(72, 129, 189)</string>
      </property>
      <property name="value" stdset="0">
       <double>10.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="13" column="6">
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string>FIRE PUNCH</string>
      </property>
     </widget>
    </item>
    <item row="15" column="7">
     <widget class="QScrollBar" name="scrlmonster">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(101, 148, 196);
color: rgb(0, 0, 0);
</string>
      </property>
      <property name="value">
       <number>2</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="8" column="1" colspan="8">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#800002;&quot;&gt;WEAPONS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="7" column="9" rowspan="3">
     <widget class="Line" name="line_7">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="12" column="6">
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>DYNAMITE</string>
      </property>
     </widget>
    </item>
    <item row="1" column="9" rowspan="2">
     <widget class="Line" name="line_8">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="7" column="0" rowspan="3">
     <widget class="Line" name="line_6">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="9">
     <widget class="Line" name="line_10">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="14" column="8">
     <widget class="QLCDNumber" name="lcdNumber_19">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(72, 129, 189)</string>
      </property>
      <property name="value" stdset="0">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>NUMBER OF TEAMS</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>WORMS PER TEAM</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QLCDNumber" name="showNumber">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(72, 129, 189)</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>3</number>
      </property>
      <property name="value" stdset="0">
       <double>2.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QScrollBar" name="scrlnbteams">
      <property name="minimumSize">
       <size>
        <width>201</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(101, 148, 196);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="minimum">
       <number>2</number>
      </property>
      <property name="maximum">
       <number>3</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>BAZOOKA</string>
      </property>
     </widget>
    </item>
    <item row="10" column="2">
     <widget class="QScrollBar" name="scrlbazooka">
      <property name="minimumSize">
       <size>
        <width>201</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(101, 148, 196);
color: rgb(0, 0, 0);
</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>GRENADE</string>
      </property>
     </widget>
    </item>
    <item row="11" column="2">
     <widget class="QScrollBar" name="scrlgrenade">
      <property name="minimumSize">
       <size>
        <width>201</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(101, 148, 196);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>PISTOL</string>
      </property>
     </widget>
    </item>
    <item row="12" column="2">
     <widget class="QScrollBar" name="scrlpistol">
      <property name="minimumSize">
       <size>
        <width>201</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(101, 148, 196);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="value">
       <number>10</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="13" column="1">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>LAND MINE</string>
      </property>
     </widget>
    </item>
    <item row="13" column="2">
     <widget class="QScrollBar" name="scrllandmine">
      <property name="minimumSize">
       <size>
        <width>201</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(101, 148, 196);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="value">
       <number>3</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="8">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="10" column="6">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>CLUSTER BOMB</string>
      </property>
     </widget>
    </item>
    <item row="5" column="8">
     <widget class="QLCDNumber" name="lcdNumberHp">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(72, 129, 189)</string>
      </property>
      <property name="value" stdset="0">
       <double>100.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="4" column="5" rowspan="3">
     <widget class="Line" name="line_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="15" column="1">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>BANANA BOMB</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="8">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#800002;&quot;&gt;GAME SETTINGS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="14" column="2">
     <widget class="QScrollBar" name="scrairstrike">
      <property name="minimumSize">
       <size>
        <width>201</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(101, 148, 196);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="value">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>HEALTH CRATES</string>
      </property>
     </widget>
    </item>
    <item row="10" column="7">
     <widget class="QScrollBar" name="scrclustebomb">
      <property name="minimumSize">
       <size>
        <width>201</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(101, 148, 196);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="value">
       <number>10</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="9" column="1" colspan="8">
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="6">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>TIME PER TURN</string>
      </property>
     </widget>
    </item>
    <item row="4" column="7">
     <widget class="QScrollBar" name="scrltimeprturn">
      <property name="minimumSize">
       <size>
        <width>201</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(101, 148, 196);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="minimum">
       <number>10</number>
      </property>
      <property name="maximum">
       <number>60</number>
      </property>
      <property name="value">
       <number>30</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QScrollBar" name="scrlhealthcrates">
      <property name="minimumSize">
       <size>
        <width>201</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(101, 148, 196);
color: rgb(0, 0, 0);
</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>25</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="7">
     <widget class="QScrollBar" name="scrlwormhp">
      <property name="minimumSize">
       <size>
        <width>201</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(101, 148, 196);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="14" column="1">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>AIRSTRIKE</string>
      </property>
     </widget>
    </item>
    <item row="15" column="2">
     <widget class="QScrollBar" name="scrlbananabomb">
      <property name="minimumSize">
       <size>
        <width>201</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(101, 148, 196);
color: rgb(0, 0, 0);
</string>
      </property>
      <property name="value">
       <number>5</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="6">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>WORMS HP</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
